src/main/webapp/timestamps.js

```diff
@@ -12,35 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const MILLISECONDS_PER_SECOND = 1000;
 const MINUTES_TO_SECONDS = 60;
 const HOURS_TO_SECONDS = 360;
 
 /**
- * Coverts a Date representing a video {@code timestamp} into a string.
+ * Converts a video {@code timestampMs} in milliseconds into a string.
```

Comment:
the 'video' doesn't matter for this util. The method only converts milliseconds to string.

Reply:
Removed "video"